models/public/face-detection-retail-0044/face-detection-retail-0044.md

```diff
@@ -0,0 +1,88 @@
+# face-detection-retail-0044
+
+## Use Case and High-Level Description
+
+Face detector based on SqueezeNet light (half-channels) as a backbone with a
+single SSD for indoor/outdoor scenes shot by a front-facing camera. The backbone
+consists of fire modules to reduce the number of computations. The single SSD
+head from 1/16 scale feature map has nine clustered prior boxes.
+
+## Example
+
+![](./face-detection-retail-0044.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP ([WIDER](http://shuoyang1213.me/WIDERFACE/))               | 83.00%                  |
+| GFlops                                                        | 1.067                   |
+| MParams                                                       | 0.588                   |
+| Source framework                                              | Caffe*                  |
```

Comment:
```suggestion
| Source framework                                              | Caffe\*                 |
```
*Hereinafter*

Reply:
@eizamaliev Sure about that change? I won't see back slash in other models.

Reply:
@elenash backslash is used to escape '*' symbol, see for example description of AlexNet

Reply:
Please, apply this change here and in *license-plate-recognition-barrier-0007*